app/views/home/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Welcome to Book Swap App</h1>
```

Comment:
We tend to use HAML instead of ERB in our Rails projects. I think it'd be good to get accustomed to it as it will be of help when you will be onboarded on other projects.

Have a look at this: https://github.com/haml/haml-rails - it even has some support for converting views from ERB to HAML, so then these `html.erb` files can be updated, see here: https://github.com/haml/haml-rails#converting-all-erb-views-to-haml-format

Reply:
Yes, from this point of view it would be really useful, I agree.
Added HAML. Thanks for links!